### DISCUSSION TITLE
Dropdowns stopped working

### DISCUSSION THREAD

**Discussion by iangullo** [excid3/tailwindcss-stimulus-components#96] [Q&A] [answered]
Hi there!

First I want to thank you for this great set of components. Effective and easy to use, they've helped me greatly to push my pet project along - a web-based platform to manage our local basketball club.

I must have done something wrong with my latest changes but Dropdowns stopped working properly - actually they are opened very vriefly and autohide immediately - and I don't know when or how to troubleshoot it.

Cand anybody provide pointers to find the issue?

**Comment by davegudge** [comment]
Hey, what setup are you using? Rails 6.1 + Turbo v7 + stimulus v2?

Does your issue look similar to the 2nd video posted on https://github.com/excid3/tailwindcss-stimulus-components/issues/63#issue-816850202 ? If so, that thread may give you some pointers.

**Reply by iangullo** [reply]
hi.

I'm using rails 6.1.4, @hotwired/turbo-rails@7.0.0-rc.3 & stimulus 2.0

**Reply by davegudge** [reply]
If you add `data: {action: "click->dropdown#toggle"}` to your `li` (dropdown options), does that resolve the issue? 

https://github.com/excid3/tailwindcss-stimulus-components/pull/65/commits/dc869013216cfeb2fcd6f157ed708c9171a011f6

**Reply by iangullo** [reply]
Nope (that was already there).
I suspect the latest yarn upgrade broke something. There have no changes in my html code but the dropdowns stopped working (they did before) and now the modals are followed as links, not overlaying the views.

**Reply by davegudge** [reply, marked as answer]
It may be worth using the [`git-bisect`](https://git-scm.com/docs/git-bisect) command to pinpoint the commit that causes the issue. If it relates to a `yarn upgrade` commit, it may be worth reverting to the commit prior to the upgrade, and then upgrade components individually, thus narrowing down the component that breaks the dropdown.
